templates/data/opensearch/index.html

```diff
@@ -0,0 +1,530 @@
+{% endblock %}
+
+{% block body_class %}
+  is-paper
+{% endblock body_class %}
+
+{% block content %}
+  <section class="p-strip is-shallow u-no-padding--bottom">
+    <div class="p-section">
+      <div class="row">
+        <div class="col-3 col-medium-2">
+          <div class="u-hide--small u-vertically-center u-aspect-ratio--16-9 u-align--center"
+               style="margin-top: -55px">
+            {{ image (
+            url="https://assets.ubuntu.com/v1/6deb3921-opensearch-logo.png",
+            alt="Opensearch",
+            width="198",
+            height="186",
+            hi_def=True,
+            loading="auto"
+            ) | safe
+            }}
+          </div>
+        </div>
+        <div class="col-9 col-medium-4">
+          <div class="p-section--shallow">
+            <h1>OpenSearch for the enterprise</h1>
+            <p>
+              Secure and automate the deployment, maintenance and upgrades of your search and analytics suite across private and public clouds.
+            </p>
+          </div>
+          <hr />
+          <a class="p-button--positive"
+             href="/contact-us"
+             aria-controls="data-spark-modal">Contact us</a>
+          <a class="p-button"
+             href="https://assets.ubuntu.com/v1/6dc3440a-Charmed%20OpenSearch%20-%20Datasheet%20-%20amended%2004%2006%202024.pdf">Datasheet</a>
+        </div>
+      </div>
+    </div>
+  </section>
+
+  <section class="p-section">
+    <hr class="p-rule is-fixed-width" />
+    <div class="row">
+      <div class="col-3 col-medium-2">
+        <h2 class="p-text--small-caps highlight">Why choose Canonical's enterprise-grade OpenSearch?</h2>
+      </div>
+      <div class="col-9 col-medium-4">
+        <ul class="p-list--divided">
+          <li class="p-list__item">
+            <h2>10 years security maintenance and support</h2>
+          </li>
+          <li class="p-list__item">
+            <h2>Open source</h2>
+          </li>
+          <li class="p-list__item">
+            <h2>Simple per node subscription</h2>
+          </li>
+          <li class="p-list__item">
+            <h2>Hybrid, multi-cloud ready</h2>
+          </li>
+        </ul>
+      </div>
+    </div>
+  </section>
+
+  <section class="p-section">
+    <div class="row--50-50">
+      <hr class="p-rule" />
+      <div class="col">
+        <h2 class="p-text--small-caps">Run OpenSearch on any public, private or hybrid cloud</h2>
+      </div>
+      <div class="col">
+        <p>
+          When you use Canonical’s OpenSearch, you get the benefit of running on top of an operating system image that’s optimised for every cloud. Run your OpenSearch on Azure, GCP, AWS, on Kubernetes or your private data centre, with OpenStack, Microcloud and VMWare.
+        </p>
+      </div>
+      <div class="p-logo-section u-hide--small">
```

Comment:
Is hiding the logos on small screens intentional? I don't believe we do this for any of the other pages